Dockerfile

```diff
@@ -1,8 +1,57 @@
-FROM geonode/django
-MAINTAINER GeoNode development team
+FROM python:2.7.9
```

Comment:
OK, we're rebasing our work on top of `geonode/django`. Thanks for the pointers.